docs/rpc/commands.md

```diff
@@ -6,10 +6,22 @@
 - [commands.proto](#commands.proto)
```

Comment:
Note: this file will be removed in the future because we should put docs in the CLI ref instead.